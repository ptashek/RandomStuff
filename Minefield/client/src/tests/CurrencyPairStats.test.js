import React from 'react';
import CurrencyPairStats from '../components/CurrencyPairStats.react';
import renderer from 'react-test-renderer';

import { PairA, TestTransactionList } from './TestData';

it('renders ok', () => {
  const tree = renderer.create(
    <CurrencyPairStats currencyPair={PairA} data={TestTransactionList.get(PairA)} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
