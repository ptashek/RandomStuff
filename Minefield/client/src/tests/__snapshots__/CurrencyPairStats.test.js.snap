exports[`test renders ok 1`] = `
<div
  className="App-Content-Main">
  Currency Pair: 
  EUR/USD
  <div>
    <div
      className="App-Content-Secondary">
      Country: 
      IE
      <ul>
        <li>
          <strong>
            Transactions:
          </strong>
           
          10
        </li>
        <li>
          <strong>
            Average rate:
          </strong>
           
          1.5
        </li>
        <li>
          <strong>
            Average size:
          </strong>
           
          100
        </li>
      </ul>
    </div>
  </div>
</div>
`;
